rclcpp/src/rclcpp/node_interfaces/node_base.cpp

```diff
@@ -31,6 +32,75 @@ using rclcpp::exceptions::throw_from_rcl_error;
 
 using rclcpp::node_interfaces::NodeBase;
 
+namespace
+{
+/// This class bundles together the lifetime of the rcl_node_t handle with the
+/// lifetime of the RCL context. This ensures that the context will remain alive
+/// for as long as the rcl_node_t handle is alive.
```

Comment:
If these comments are meant to be doxygen, please use this style which is prevalent throughout the project:

```
/// Brief, single-line description.
/**
 * Everything else...
 */
```

If not, please just use `//`.

Reply:
Fixed in c5ff0f4